account.sh

```diff
@@ -0,0 +1,217 @@
+#!/bin/bash
+
+########## VARIABLES FOR ACTIVE USER ##########
+### loginUser() fills these in
+userId=0
+name=""
+email=""
+orders=()
+
+################ GETTER/SETTERS ################
+### GET functions require arguments noted
+
+getId() {   # $1:Email
+    for (( i=1; i < $(jq '.users | length' customers.json); i++ )); do  # starts at 1 since users[0] contains guest checkouts
```

Comment:
: i < : syntax error: operand expected (error token is "< "). Looked at this line briefly and it seems fine, not sure what's causing the error

Reply:
Does it pop up after you enter email and name? Works fine for me, but I'll see if it happens when I try it on my VM instead on Git Bash. Wonder if that `i` needs to be `$i`

Reply:
> : i < : syntax error: operand expected (error token is "< "). Looked at this line briefly and it seems fine, not sure what's causing the error

Does it pop up after you enter email and name? Works fine for me, but I'll see if it happens when I try it on my VM instead on Git Bash. Wonder if that i needs to be $i

Reply:
Yep, after entering an email.
![Capture](https://user-images.githubusercontent.com/36974628/104472655-29cbc500-558a-11eb-88b6-80b9d8af1a31.JPG)

Reply:
> Yep, after entering an email, also get a jq not found, which i guess is why it throws the error.
> ![Capture](https://user-images.githubusercontent.com/36974628/104472655-29cbc500-558a-11eb-88b6-80b9d8af1a31.JPG)

Yeah, I think that's why. I got the same syntax error once I uninstalled jq